suite-native/module-accounts-import/src/components/SelectableNetworkList.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import { A, pipe } from '@mobily/ts-belt';
+
+import { Box, Card, Text, VStack } from '@suite-native/atoms';
+import {
+    Network,
+    networks,
+    NetworkSymbol,
+    getMainnets,
+    getTestnets,
+} from '@suite-common/wallet-config';
+import { mainnetsOrder, testnetsOrder } from '@suite-native/config';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { SelectableNetworkItem } from './SelectableNetworkItem';
+
+type SelectableAssetListProps = {
+    onSelectItem: (networkSymbol: NetworkSymbol) => void;
+};
+
+const listItemTitleStyle = prepareNativeStyle(utils => ({
+    marginBottom: utils.spacings.medium,
+}));
+
+const mainnets: Network[] = getMainnets();
+const testnets: Network[] = getTestnets();
+
+const mapAndSortNetworkItems = (networkItems: Network[], networkOrder: NetworkSymbol[]) =>
+    pipe(
+        networkItems,
+        A.map(network => ({
+            cryptoCurrencySymbol: network.symbol,
+            cryptoCurrencyName: networks[network.symbol].name,
+            iconName: network.symbol,
+            order: networkOrder.includes(network.symbol)
+                ? networkOrder.indexOf(network.symbol)
+                : Number.MAX_SAFE_INTEGER,
+        })),
+        A.sort((a, b) => (a.order > b.order ? 1 : -1)),
+    );
+
+export const SelectableNetworkList = ({ onSelectItem }: SelectableAssetListProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const getSelectableNetworkItems = (networkItems: Network[], networkOrder: NetworkSymbol[]) => {
+        const sortedNetworkItems = mapAndSortNetworkItems(networkItems, networkOrder);
+
+        return sortedNetworkItems.map(({ cryptoCurrencySymbol, cryptoCurrencyName, iconName }) => (
+            <SelectableNetworkItem
+                key={cryptoCurrencySymbol}
+                iconName={iconName}
+                cryptoCurrencyName={cryptoCurrencyName}
+                cryptoCurrencySymbol={cryptoCurrencySymbol}
+                onPress={() => onSelectItem(cryptoCurrencySymbol)}
```

Comment:
If you will make `SelectableNetworkItem` to call onPress prop with argument `onPress: cryptoCurrencySymbol => void` you can get rid of this anonym function that will possible cause rerenders `onPress={onSelectItem}`